docs/content/integration/openid-connect/odoo/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Odoo"
+description: "Integrating Odoo with the Authelia OpenID Connect 1.0 Provider."
+summary: ""
+date: 2024-03-24T09:09:0+01:00
+draft: false
+images: []
+weight: 620
+toc: true
+community: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Tested Versions
+
+* [Authelia]
+  * [v4.38.6](https://github.com/authelia/authelia/releases/tag/v4.38.6)
+* [Odoo]
+  * [17.0](https://github.com/odoo/odoo/tree/17.0)
+
+## Before You Begin
+
+{{% oidc-common %}}
+
+### Assumptions
+
+This example makes the following assumptions:
+
+* __Application Root URL:__ `https://odoo.example.com/`
+* __Authelia Root URL:__ `https://auth.example.com/`
+* __Client ID:__ `client_id_odoo`
+
+## Configuration
+
+### Authelia
+
+The following YAML configuration is an example __Authelia__
+[client configuration](../../../configuration/identity-providers/openid-connect/clients.md) for use with [Odoo]
+which will operate with the above example:
+
+```yaml
+identity_providers:
+  oidc:
+    clients:
+      - client_id: 'client_id_odoo'
+        client_name: 'odoo.example.com'
+        public: 'true'
+        redirect_uris:
+          - 'https://odoo.example.com/auth_oauth/signin'
+        scopes:
+          - 'openid'
+          - 'email'
+          - 'profile'
+        response_types:
+          - 'token'
+        authorization_policy: 'one_factor'
+        jwks:
+          - key_id: 'rsa2048'
+            key: {{ secret "/config/secrets/oidc/jwks/public.pem" | mindent 14 "|" | msquote }}
```

Comment:
Where does the client JWK come from? Doesn't seem described in the following sections.

Reply:
The file was generating using this doc: https://www.authelia.com/reference/guides/generating-secure-values/#generating-an-rsa-keypair and enabled using the (experimental ?) filters as documented here : https://www.authelia.com/configuration/methods/files/#file-filters.

If you want, I can add some note about it.

Reply:
Well the JWK is meant to be utilized by Odoo in this instance, how does it use it? Seems from their docs they only support a client secret maybe or how are you configuring its credentials?

Reply:
You're right. I didn't used this JWK in the Odoo configuration... It seems to not require it.

The only shared secret, is the client_id.